### PR TITLE
Add Iflight Beastf7 AIO

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-iflight-beastf7AIO.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+The board has a built-in voltage and current sensors.
+
+The correct battery monitor parameters are:
+
+-    :ref:`BATT_MONITOR<BATT_MONITOR>` =  4
+-    :ref:`BATT_VOLT_PIN<BATT_VOLT_PIN>` = 12
+-    :ref:`BATT_VOLT_MULT<BATT_VOLT_MULT>` ~ 10.9
+-    :ref:`BATT_CURR_PIN<BATT_CURR_PIN>` = 13
+-    :ref:`BATT_AMP_PERVLT<BATT_AMP_PERVLT>` ~ 28.5
```
Comment (andyp1per) [inline common/source/docs/common-iflight-beastf7AIO.rst:100]: This was wrong in my original definition it should be 100

**Review by andyp1per — CHANGES_REQUESTED** [review]

File: common/source/docs/common-iflight-beastf7AIO.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-iflight-beastf7AIO.rst:100]: you should set the log backend to 4 also in the defaults

Comment (Hwurzburg) [reply common/source/docs/common-iflight-beastf7AIO.rst:100]: so its a 300A sensor?

Comment (Hwurzburg) [reply common/source/docs/common-iflight-beastf7AIO.rst:100]: its specs say 200mv/10a which would be 50 for the amp/volt

Comment (andyp1per) [reply common/source/docs/common-iflight-beastf7AIO.rst:100]: > you should set the log backend to 4 also in the defaults

Not needed - this is done automatically by specifying flash logging

Comment (andyp1per) [reply common/source/docs/common-iflight-beastf7AIO.rst:100]: > so its a 300A sensor?

The docs say the "current scale: 200", but this value is what gives me believable results

Comment (Hwurzburg) [reply common/source/docs/common-iflight-beastf7AIO.rst]: okay, if you have measured that 100 is the correct value...will change in both docs...that implies a 300A current sensor range..which I will change also...

**Review by andyp1per — APPROVED** [review]

**Pull request merged**